custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/r_predictor.py

```diff
@@ -104,3 +103,56 @@ def predict(self, input_filename):
                 logger.error(error_message)
                 raise DrumCommonException(error_message)
         return py_data_object
+
+    def predict_unstructured(self, **kwargs):
```

Comment:
As discussed, for now we'll go with directly passing **kwargs to the python's R proxy function.
Would be great to avoid this transformation alltogehter because string builders are hard when it comes to various data. For example this one lacks quote escaping. We'll see if we can find a more durable solution.

For now this transformation would be used for arbitrary data, for example query args or web headers.

Reply:
**DONE**